### PR TITLE
Clean up READMEs in prep for cert

### PR DESCRIPTION
**Pull request by arsalansufi**
## Overview

Issue link: https://github.com/votingworks/vxsuite/issues/5028

This is what prepping for cert looks like haha. Just taking a pass through our fairly out-of-date READMEs and updating. I'm trimming things down where I can since we're also going to have our TDP documentation as a new source of truth. I actually think that that documentation will work not only for cert but for onboarding engineers as well!

### CODE REVIEW

File: apps/central-scan/backend/README.md [hunk 1]
```diff
@@ -1,76 +1,5 @@
...
-
-```sh
-cd apps/central-scan/frontend
-pnpm start
-```
-
-The commands to run the service below assume you'll be running them in a
-frontend, not this service directly.
-
-## Mock Scanning
```
Comment (arsalansufi) [inline apps/central-scan/backend/README.md:19]: Moved these instructions to the frontend README - generally sticking with the pattern of the frontend README being the landing page for each app, with the backend README pointing to that

File: apps/central-scan/frontend/README.md [hunk 2]
```diff
@@ -1,25 +1,65 @@
...
 
 ## Testing
 
 ```sh
 pnpm test
 ```
+
+## Development Tips
+
+### Mock Scanning
```
Comment (arsalansufi) [inline apps/central-scan/frontend/README.md:26]: As noted above, moved from the backend README

File: apps/mark-scan/frontend/CONTRIBUTING.md [hunk 3]
```diff
@@ -1,10 +0,0 @@
-# Contributing to the VotingWorks Ballot Marking Device
```
Comment (arsalansufi) [inline apps/mark-scan/frontend/CONTRIBUTING.md:1]: Relying on our existing top-level CONTRIBUTING.md instead

File: docs/development/best_practices/rust.md [hunk 4]
```diff
@@ -1,4 +1,87 @@
-# Rust Development
+## Rust Best Practices
```
Comment (arsalansufi) [inline docs/development/best_practices/rust.md:1]: Main edits here are:

1. Mirroring the typescript.md
2. Shuffling the order of things, moving the "Tooling" section up and the "Learning" section down, to keep everything that SLI/EAC cares about grouped together
3. Updating the style guide permalink

File: docs/development/best_practices/typescript.md [hunk 5]
```diff
@@ -6,8 +6,9 @@ and parts of [Airbnb JavaScript Style Guide](https://airbnb.io/javascript/),
 with most of it enforced by ESLint rules. This document covers some of the best
 practices that are not automatically enforced by ESLint. We believe our code to
 meet all of the recommendations in the Google TypeScript Style Guide published
-as of April 27, 2023, an archived copy of the guide on that date is kept
-[here](./04-27-2023-Google-TypeScript-Style-Guide.pdf) for reference.
+as of November 21, 2024. A link to the guide as of that date is kept
+[here](https://github.com/google/styleguide/blob/4153bf1f8c172fcb58593193238b0a928e58e61e/tsguide.html)
```
Comment (arsalansufi) [inline docs/development/best_practices/typescript.md:10]: No more PDF yay!

File: CONTRIBUTORS.md [hunk 6]
```diff
@@ -1,23 +1,30 @@
 # Contributors
 
-This document will evolve as we make sure to include all the
-contributors to the VotingWorks system.
+This document will evolve as we make sure to include all the contributors to the
+VotingWorks system.
```
Comment (arsalansufi) [inline CONTRIBUTORS.md:4]: Looks like my editor is auto-formatting to an 80-char line width

File: CONTRIBUTORS.md [hunk 7]
```diff
@@ -1,23 +1,30 @@
...
 
 In Calhoun County, Mississippi: Carlton Baker
 
 In Woodstock, New Hampshire: Judy Welch and Ken Chapman
 
-In Moultonborough, New Hampshire: Julia Marchand and Paul Punturieri
\ No newline at end of file
+In Moultonborough, New Hampshire: Julia Marchand and Paul Punturieri
+
+## Security Architecture
```
Comment (arsalansufi) [inline CONTRIBUTORS.md:25]: Moved from the vxsuite-complete-system README.md (which I'll be updating next), to keep all contributors consolidated in one place

**Review by benadida — APPROVED** [review]

**Review by carolinemodic — APPROVED** [review]

**Pull request merged**